mem/mem.c

```diff
@@ -2090,6 +2082,13 @@ static void pfx_ctrace(const char *format, ...)
 
 #if !defined(USE_SYS_ALLOC) && !defined(COMDB2MA_OMIT_BMEM)
 /* COMDB2 BLOCKING MEMORY ALLOCATOR { */
+
+/* the pthread key is shared amongst all blocking allocators */
```

Comment:
I specifically placed these lines in "static code" section. Is this causing a compiler warning?

Reply:
Yes, i put them back where you had them, and surrounded them with appropriate ifdef.
Thanks